cloud-sql-proxy/cloud-sql-proxy.sh

```diff
@@ -259,12 +310,20 @@ function main() {
       else
         echo "Service hive-metastore is not enabled"
       fi
-      systemctl stop mysql
-      systemctl disable mysql
+      if ( systemctl is-enabled --quiet mysql ); then
+        systemctl stop mysql
+        systemctl disable mysql
+      fi
     fi
     install_cloud_sql_proxy
     if [[ $enable_cloud_sql_metastore = "true" ]]; then
-      configure_sql_client
+      if [[ "${HOSTNAME}" == "${DATAPROC_MASTER}" ]]; then
+        #initialize metastore db instance and set hive.metastore.warehouse.dir on master 0
```

Comment:
nit: comment style here and throughout. Space after '#', capitalize first word, period at end